### ISSUE TITLE
Implement Seed Animation

### ISSUE THREAD

**Issue by supreme2580** [realm-of-ra/mancala#182]
Implement seed animation using framer js or any other js animation library for seed movements, seed capture and end game.

I recommend you work from update-contract branch as the query there contains extra fields you may need namely previous and current pit for each seed.

We already have a working version on the develop branch, so you can checkout that to play and see how it looks/is, then you can see why we want to animate seeds from the update-contract branch

Nobody will be assigned, the first PR with this feature working claims this.

**Comment by Iwueseiter** [comment]
Hi @supreme2580 can I work on this?

**Comment by supreme2580** [comment]
Yeah, it is open to everybody, we'll take the first PR that works

**Comment by Iwueseiter** [comment]
Alright
